.pre-commit-config.yaml

```diff
@@ -36,7 +36,6 @@ repos:
     rev: v2.6.2
     hooks:
       - id: prettier
-        language_version: system
```

Comment:
I think we should either remove the requirement to have a system nodejs installation for pre-commit to work or add a contributing guide (xref https://github.com/pangeo-data/xbatcher/issues/32) that includes a pre-commit explainer.